code/__HELPERS/unsorted.dm

```diff
@@ -1366,6 +1366,13 @@ proc/find_holder_of_type(var/atom/reference,var/typepath) //Returns the first ob
 /proc/print_runtime(exception/e)
 	world.log << "[time_stamp()] Runtime detected\n[e] at [e.file]:[e.line]\n [e.desc]"
 
+/proc/transfer_fingerprints(atom/A,atom/B)
+	if(!A || !B)
+		return
+	B.fingerprints = A.fingerprints
```

Comment:
@DeityLink since these variables are lists, you're not copying the list, you're literally setting B's fingerprints var equal to A's, if you were to modify them on either B or A, the other atom would get the modified prints too.